db/schema.rb

```diff
@@ -0,0 +1,36 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2019_09_13_104950) do
+
+  create_table "events", force: :cascade do |t|
+    t.text "description"
+    t.datetime "date"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "creator_id"
+  end
+
+  create_table "invitations", force: :cascade do |t|
+    t.integer "attendee_id"
+    t.integer "attended_event_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
```

Comment:
This is the requirement for expected behaviour:
 - It has functional login page (authentication WITHOUT password, just providing a user email)

Bear in mind that names are not necessarily unique and you have no control on whether the user inserts just first name, last name, full name, username or even nickname. Think about how *not* guaranteeing uniqueness on this field can affect your application? If its not unique then you won't be able to log an user in, and a `name` field is not a good fit for uniqueness, as we all know we can have people with identical names. 

If you will use name field, I'd suggest you use `username` instead, as it makes more sense to guarantee uniqueness with it. 

[`REQUIRED`]